tests/integration/test_cloudformation.py

```diff
@@ -628,6 +628,27 @@ def handler(event, context):
       BucketName: cf-prd-{id}
 """ % TEST_AWS_ACCOUNT_ID
 
+TEST_TEMPLATE_20 = """
+AWSTemplateFormatVersion: 2010-09-09
+Description: Test template
+Resources:
+  LambdaFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Runtime: nodejs10.x
+      Handler: index.handler
+      Role: arn:aws:iam::000000000000:role/lambda-ex
```

Comment:
nitpick: Can we add a placeholder here, and insert the role ARN based on what is returned from the `create_role(..)` call below?